packages/mds-ingest-service/service/validators.ts

```diff
@@ -64,31 +64,27 @@ const telemetrySchema = Joi.object<TelemetryDomainModel>()
   })
   .unknown(false)
 
-export const {
-  validate: validateTelemetryDomainModel,
-  isValid: isValidTelemetryDomainModel
-} = schemaValidator<DeviceDomainModel>(telemetrySchema)
+export const { validate: validateTelemetryDomainModel, isValid: isValidTelemetryDomainModel } =
+  schemaValidator<DeviceDomainModel>(telemetrySchema)
 
-export const {
-  validate: validateEventDomainModel,
-  isValid: isValidEventDomainModel
-} = schemaValidator<DeviceDomainModel>(
-  Joi.object<EventDomainModel>()
-    .keys({
-      device_id: Joi.string().uuid().required(),
-      provider_id: Joi.string().uuid().required(),
-      timestamp: Joi.number().required(),
-      event_types: Joi.array()
-        .valid(Joi.string().valid(...VEHICLE_EVENTS))
-        .required(),
-      vehicle_state: Joi.string().valid(...VEHICLE_STATES),
-      telemetry_timestamp: Joi.number().allow(null),
-      telemetry: telemetrySchema.allow(null),
-      trip_id: Joi.string().uuid().allow(null),
-      service_area_id: Joi.string().uuid().allow(null)
-    })
-    .unknown(false)
-)
+export const { validate: validateEventDomainModel, isValid: isValidEventDomainModel } =
+  schemaValidator<DeviceDomainModel>(
+    Joi.object<EventDomainModel>()
+      .keys({
+        device_id: Joi.string().uuid().required(),
+        provider_id: Joi.string().uuid().required(),
+        timestamp: Joi.number().required(),
+        event_types: Joi.array()
+          .valid(Joi.string().valid(...VEHICLE_EVENTS))
+          .required(),
+        vehicle_state: Joi.string().valid(...VEHICLE_STATES),
+        telemetry_timestamp: Joi.number().allow(null),
+        telemetry: telemetrySchema.allow(null),
+        trip_id: Joi.string().uuid().allow(null),
+        service_area_id: Joi.string().uuid().allow(null)
+      })
+      .unknown(false)
+  )
```

Comment:
Different version of `Prettier` installed? Why all the formatting changes?

Reply:
Saw this as well. Are there standard formatting settings (tabs/spaces/etc) I should be abiding by? I can make sure to use it and recommit.

Reply:
We have rules defined in `.prettierrc.json` and `.eslintrc.json` which define our formatting standards. If you use VSCode and install the extensions recommended by the `.vscode/extensions.json` file in this repo, you ought to auto-magically have all the formatting rules applied. Also, I think that they're typically applied as part of a pre-commit linting hook.